scripts/run_all_benchmarks.sh

```diff
@@ -41,5 +226,12 @@ for pallet_name in "${CUSTOM_PALLETS[@]}"; do
   repeat=10
   template=${PROJECT}/.maintain/frame-weight-template.hbs
   output=${PROJECT}/pallets/${pallet_name}/src/weights.rs
-  run_benchmark pallet_${pallet_name} ${steps} ${repeat} ${output} ${template} || exit_err
+  run_benchmark ${pallet_name} ${steps} ${repeat} ${output} ${template} || exit_err
 done
+
+if [[ -n "${OVERHEAD}" ]]
+then
+  echo "Running extrinsic and block overhead benchmark"
+  echo " "
+  ${BENCHMARK} overhead --execution=wasm --wasm-execution=compiled --weight-path=runtime/common/src/weights --chain=dev --warmup=10 --repeat=100 || exit_err
+fi
```

Comment:
I think we can do this change in another PR since it is not directly related to private graph and it's over improvement for benchmarking. 
Created an issue to track this https://github.com/LibertyDSNP/frequency/issues/1231